test/onnx/test_utility_funs.py

```diff
@@ -59,6 +59,23 @@ def test_validate_dynamic_axes_invalid_input_output_name(self):
         assert "Provided key invalid_name2 for dynamic axes is not a valid input/output name" in messages
         assert len(messages) == 2
 
+    @skipIfUnsupportedMinOpsetVersion(11)
```

Comment:
is test in pytorch_onnx_onnxruntime also needed?

Reply:
i think it's fine, the code path is covered by current tests in test_pytorch_onnx_onnxruntime.py. This is an additional test to ensure the nodes do appear in onnx as slices instead of SplitToSequence.